src/net/sourceforge/kolmafia/session/JuneCleaverManager.java

```diff
@@ -0,0 +1,45 @@
+package net.sourceforge.kolmafia.session;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.sourceforge.kolmafia.preferences.Preferences;
+
+public class JuneCleaverManager {
```

Comment:
as this class only has static methods, can you add a private constructor to get rid of the coverage warning?